erpnext/hr/doctype/shift_type/shift_type.py

```diff
@@ -79,7 +79,7 @@ def mark_absent_for_dates_with_no_attendance(self, employee):
 				mark_attendance(employee, date, 'Absent', self.name)
 
 	def get_assigned_employee(self, from_date=None, consider_default_shift=False):
-		filters = {'date':('>=', from_date), 'shift_type': self.name, 'docstatus': '1'}
+		filters = {'start_date':('>=', from_date), 'shift_type': self.name, 'docstatus': '1'}
```

Comment:
this woud be the right one... this function looks for employees assigned to this shift after a given date.
```suggestion
		filters = {'end_date':('>=', from_date), 'shift_type': self.name, 'docstatus': '1'}
```

Reply:
Yeah, it is done intentionally because if there is no end_date it means that the shift is ongoing.

Reply:
No, i mean `'start_date':('>=', from_date)` will not work at all... This function returns all employees assigned to this shift...

also, if some shifts do not have end_date, even my suggestion `'end_date':('>=', from_date)` would not work.

The right one would be as follows:
`filters = {'shift_type': self.name, 'docstatus': '1'}`

can you please make the change accordingly.

Reply:
done